tests/unittests/utils/configuration/test_settings.py

```diff
@@ -158,3 +169,16 @@ def test_attributes_whose_key_is_absent_return_expected_default(
         config = UDCTokenSettings(minimal_yaml_dict)
         MISSING = object()
         assert getattr(config, key, MISSING) == expected
+
+    def test_balance_per_node_bigger_than_max_funding(self, minimal_yaml_dict):
```

Comment:
```suggestion
    def test_balance_per_node_must_not_be_greater_than_max_funding(self, minimal_yaml_dict):
```

Otherwise, it sounds like your assertin that balance_per_node should be greater than token max_funding.